components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -917,6 +919,8 @@ void updateConsumerApplication(OAuthConsumerAppDTO consumerAppDTO, boolean enabl
             }
             oauthappdo.setRequestObjectEncryptionMethod(requestObjectEncryptionMethod);
             oauthappdo.setRequirePushedAuthorizationRequests(consumerAppDTO.getRequirePushedAuthorizationRequests());
+            oauthappdo.setSubjectTokenEnabled(consumerAppDTO.isSubjectTokenEnabled());
```

Comment:
From a different PR, can you please re-name this variable  oauthappdo>  oauthAppDo

Reply:
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2468